### PR TITLE
WT-5719 Add quotes to the metadata string. Modify example to use special

### PR DESCRIPTION
**Pull request by sueloverso**
characters.

### CODE REVIEW

**Comment by keitharnoldsmith** [comment]
Testing if I can comment in conversation.

File: examples/c/ex_backup_block.c [hunk 1]
```diff
@@ -268,8 +268,8 @@ take_full_backup(WT_SESSION *session, int i)
     } else
         hdir = home_incr;
     if (i == 0) {
-        (void)snprintf(
-          buf, sizeof(buf), "incremental=(granularity=1M,enabled=true,this_id=ID%d)", i);
+        (void)snprintf(buf, sizeof(buf),
+          "incremental=(granularity=1M,enabled=true,this_id=\"ID%d:%d.%d\")", i, i, i);
```
Comment (ddanderson) [inline examples/c/ex_backup_block.c]: I think this code represents a valid test for the problem, but I wonder if the special cases might be better handled in the test suite.  I think `test_backup11.py` has some error testing for incremental backup, so trying out special characters there might be better?

Comment (sueloverso) [reply examples/c/ex_backup_block.c]: Good idea. I'll make that fix.

Comment (sueloverso) [reply examples/c/ex_backup_block.c]: Fixed.

**Comment by sueloverso** [comment]
@keitharnoldsmith I tagged you as a reviewer.

**Comment by ddanderson** [comment]
@sueloverso, thanks - LGTM.

**Pull request merged**